### PR TITLE
Allow biotope/element to be bundeled

### PR DESCRIPTION
**Pull request by SheepFromHeaven**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -87,6 +87,16 @@ module.exports = {
...
     noEmitOnErrors: true,
-    concatenateModules: true
+    concatenateModules: true,
+    splitChunks: {
+      chunks: 'all',
+      name: 'resources/js/vendor/biotope-element',
+      cacheGroups: {
+        vendors: {
+          test: /[\\/]@biotope[\\/]element[\\/]/,
+          priority: -10
```
Comment (tiagomapmarques) [inline webpack.config.js]: Q: `-10` seems like a magic number. Is there a meaning to it, or is it just meant to be a low number?

Comment (tiagomapmarques) [reply webpack.config.js]: Bump

Comment (timomayer) [reply webpack.config.js]: @SheepFromHeaven can you answer the open question so we can merge?

Comment (tiagomapmarques) [reply webpack.config.js]: It's just a question, I'm ok merging without any changes.
The intent was just to understand the value's meaning 😄

Comment (SheepFromHeaven) [reply webpack.config.js]: @tiagomapmarques this gives the user the possibility to add custom cache groups himself which bundle biotope/element. Without a low priority, our cachegroup would always be prefered.

Comment (SheepFromHeaven) [reply webpack.config.js]: => so just a low number

**Pull request merged**